src/systems/camera-system.js

```diff
@@ -213,7 +219,7 @@ export class CameraSystem {
     this.mode = NEXT_MODES[this.mode] || 0;
   }
 
-  inspect(o, distanceMod, temporarilyDisableRegularExit) {
+  inspect(o, distanceMod, temporarilyDisableRegularExit, optionalAlternativeToNotHide) {
```

Comment:
Probably better to replace `optionalAlternativeToNotHide` with an offset instead. Did not do it in this PR because it was slightly more work, but might followup and do it later or leave a comment about it in the code.